### DISCUSSION TITLE
yaml setup with multiple temp and dest drives

### DISCUSSION THREAD

**Discussion by cjcoopertag** [swar/Swar-Chia-Plot-Manager#970] [Q&A]
So in yaml if I set up just two temp and two destination drives I have no issue plotting.  It's when I list all of my temp and dest drives I am unable to run and receive the response shown when I try to start it.  What am I missing here?

I am running a Threadripper 3990X 64core.  
![Screenshot 2021-05-28 094313](https://user-images.githubusercontent.com/83853381/120001696-aa1f1f80-bf99-11eb-9baf-167e11a9d3a2.png)
![Screenshot 2021-05-28 094403](https://user-images.githubusercontent.com/83853381/120001698-aab7b600-bf99-11eb-9dfb-19cf24749daf.png)

**Comment by cjcoopertag** [comment]
I got it figured out

**Reply by Dschijn** [reply]
please explain :)

**Reply by papa41652** [reply]
would like to know how you did it.
